go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -3,14 +3,18 @@ package services
 import (
 	"net"
 
+	discovery "k8s.io/api/discovery/v1"
+
 	globalconfig "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 )
 
 // hasHostEndpoints determines if a slice of endpoints contains a host networked pod
-func hasHostEndpoints(endpointIPs []string) bool {
-	for _, endpointIP := range endpointIPs {
-		if IsHostEndpoint(endpointIP) {
-			return true
+func haveEndpointsAHostAddress(endpoints []discovery.Endpoint) bool {
```

Comment:
nit: fix naming & doc

Reply:
shouldn't it be more like `haveEndpointsAsHostAddress` but I see it backwards. Why not keep the original name?

Reply:
I think I get it now. But it does proof the name is confusing :P

Reply:
How about `doEndpointsIncludeHostAddresses`?